b_cfn_elasticsearch_cloner/cloner_source/index.py

```diff
@@ -51,52 +73,69 @@ def handler(event: Dict[Any, Any], context: Any) -> None:
     :return: No return.
     """
 
-    logger.info("Starting processing DynamoDB events.")
+    logger.info('Starting processing DynamoDB events.')
+
+    use_embeddings = bool(
+        SAGEMAKER_EMBEDDINGS_KEY and SAGEMAKER_ENDPOINT_NAME and EMBEDDINGS_KEY)
+
+    if use_embeddings:
+        logger.info(f'Performing embeddings.')
```

Comment:
Are embeddings being computed here? As I see - no. Place such logging where the described action is actually taking place, as it will allow easier future debugging.